mesonbuild/modules/qt.py

```diff
@@ -556,14 +556,14 @@ def compile_translations(self, state: 'ModuleState', args: T.Tuple, kwargs: 'Com
             else:
                 outdir = state.subdir
             cmd = [self.tools['lrelease'], '@INPUT@', '-qm', '@OUTPUT@']
-            lrelease_kwargs = {'output': '@BASENAME@.qm',
-                               'input': ts,
-                               'install': kwargs['install'],
-                               'install_tag': 'i18n',
-                               'build_by_default': kwargs['build_by_default'],
-                               'command': cmd}
-            if install_dir is not None:
-                lrelease_kwargs['install_dir'] = install_dir
+            lrelease_kwargs: T.Dict[str, T.Any] = {
```

Comment:
I think this could be `T.Dict[str, TYPE_var]` (or even better a typed dict)

Reply:
CustomTarget currently takes `T.Dict[str, T.Any]` for its initializer, so we'd have to change that too. I've got another series built on this that is moving towards not passing a dictionary into CustomTarget, but keyword arguments, since all the argument parsing should be happening in the interpreter.

Reply:
Fine by me then.